### DISCUSSION TITLE
Créer un chatbot avec Flutter en 6 lignes de code

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#199] [Announcements]
# Créer un chatbot avec Flutter en 6 lignes de code

Créez facilement un chatbot interactif dans votre application Flutter  en mettant l'accent sur l'importance du contexte dans la communication utilisateur.

https://zerofiltre.tech/articles/289

**Comment by imphilippesimo** [comment]
Le code n'est pas correctement indenté à certains endroits. Merci de le faire

**Comment by imphilippesimo** [comment]
Concernant le code source abordé dans les articles, il existe une procédure claire dont vous êtes au courant.
Merci de la respecter en regardant la [section 2.2 de cet article](https://zerofiltre.tech/articles/181)

**Reply by arolleaguekeng** [reply]
en ce qui concerne l'indentation c'est lorsque je colle le code que je copie dans mon IDE et que je colle sur la plateforme qu'il n'est plus indenté. le code que je copie dans monn IDE respecte les bonnes pratiques de DART en ce qui concer l'indentation et les syntaxe

**Reply by imphilippesimo** [reply]
Je vous propose d'utiliser 8 espaces en lieu et place de la tabulation. Nous rencontrons des soucis avec la tabultation sur l'éditeur Zerofiltre

**Comment by imphilippesimo** [comment]
J'insiste sur la nécessité de relire cette procédure: 

https://github.com/Zerofiltre-Courses/articles/edit/main/README.md

**Reply by arolleaguekeng** [reply]
en ce qui concerne cette procédure elle a déja été réalisé avant la soummission de l'artice

**Reply by imphilippesimo** [reply]
La pull request a été acceptée. Merci de mettre à jour les liens dans l'article.

**Comment by arolleaguekeng** [comment]
je viens d'apporter les modifications que vous avez soulevé

Le lun. 18 mars 2024 à 21:17, Philippe SIMO GUEMKAM <
***@***.***> a écrit :

> Je vous propose d'utiliser 8 espaces en lieu et place de la tabulation.
> Nous rencontrons des soucis avec la tabultation sur l'éditeur Zerofiltre
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zero-filtre/zerofiltre-blog/discussions/199#discussioncomment-8832743>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWIF4U6VVXE53DBA7N2B3IDYY5DU3AVCNFSM6AAAAABE2FGWECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DQMZSG42DG>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 
Cordially
*AGUEKENG Arolle Dubois*

Software Developer Consultant | Full-Stack Web and Mobile Developer | *Devpea
<https://devpea.tech> Founder | Lead*

Look at my *portfolio <https://pharaon.live>*
Follow me on *Linkedin <https://www.linkedin.com/in/aguekeng/>*
